packages/atomic/src/components/search/facets/atomic-category-facet/e2e/atomic-category-facet.e2e.ts

```diff
@@ -0,0 +1,30 @@
+import {test, expect} from './fixture';
+
+test.describe('when clicking facet search "More matches for"', () => {
+  test.beforeEach(async ({facet}) => {
+    await facet.load({story: 'low-facet-values'});
+  });
+  test('should display an increasing number of matches', async ({facet}) => {
+    await facet.getFacetSearch.click();
+    await facet.getFacetSearch.pressSequentially('p');
+    await expect
+      .poll(async () => {
+        return await facet.getFacetValue.count();
+      })
+      .toBeGreaterThanOrEqual(2);
+
+    await facet.facetSearchMoreMatchesFor.click();
+    await expect
+      .poll(async () => {
```

Comment:
From what I gather, the reason why we're polling here is that we don't have a visual indicator (e.g., a loading spinner) that the network request is in progress. Otherwise we could base our tests on the visibility of that element.

I think it would be better if we had something like that, although it probably falls outside the scope of this PR.